src/plugins/custom_integrations/server/plugin.test.ts

```diff
@@ -136,9 +136,22 @@ describe('CustomIntegrationsPlugin', () => {
           uiInternalPath:
             'https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/branch/index.html',
           isBeta: false,
-          icons: [{ type: 'svg' }],
+          icons: [{ type: 'svg', src: undefined }],
           categories: ['elastic_stack', 'custom', 'language_client'],
         },
+        {
+          id: 'placeholder.esf',
+          title: 'AWS Serverless Application Repository',
+          description:
+            'Collect logs using AWS Lambda application available in AWS Serverless Application Repository.',
+          type: 'ui_link',
+          shipper: 'placeholders',
+          uiInternalPath:
```

Comment:
I think we probably need to add a new field to the the integration as internal path seems not correct here, `uiExternalLink` or something like this